src/odemis/driver/smaract.py

```diff
@@ -291,119 +295,203 @@ def __str__(self):
 
 class SmarPod(model.Actuator):
     
-    def __init__(self, name, role, locator, ref_on_init=False, actuator_speed=0.1,
-                 axes=None, **kwargs):
+    def __init__(self, name: str, role: str, locator: str, hwmodel: int=10001,
+                 axes: dict=None, ref_on_init: bool=False,
+                 speed: float=1e-3, accel: float=1e-3,
+                 hold_time=float("inf"), **kwargs):
         """
         A driver for a SmarAct SmarPod Actuator.
-        This driver uses a DLL provided by SmarAct which connects via
+        This driver uses the SmarPod DLL provided by SmarAct which connects via
         USB or TCP/IP using a locator string.
 
         name: (str)
         role: (str)
         locator: (str) Use "fake" for a simulator.
-            For a real device, MCS controllers with USB interface can be addressed with the
+            For a real device, MCS2 controllers with USB interface can be addressed with the
             following locator syntax:
-                usb:id:<id>
-            where <id> is the first part of a USB devices serial number which
-            is printed on the MCS controller.
-            If the controller has a TCP/IP connection, use:
-                network:<ip>:<port>
-        ref_on_init: (bool) determines if the controller should automatically reference
-            on initialization
-        actuator_speed: (double) the default speed (in m/s) of the actuators
+                usb:sn:<serialnumber>
+            where <serialnumber> is the serial number of an MCS2 controller.
+            Be aware that on Linux, a udev rules must be added to disable the standard
+            tty driver from connecting to the device.
+            If the controller has a TCP/IP connection, use one of:
+                network:<ipv4>
+                network:sn:<serialnumber>
+        hwmodel (int): the hardware model code (typically between 10000 and 10100)
```

Comment:
For the other arguments in this function the docstring format is `name: (type)` instead of `name (type):`

Reply:
I've updated the doctring to either not have the type anymore (for the types which are already fully defined in the type annotation), or formatted as `name (type):`.